app/presenters/tree_node_builder.rb

```diff
@@ -186,7 +194,8 @@ def tooltip(tip)
   end
 
   def node_icon(icon)
-    ActionController::Base.helpers.image_path("100/#{icon}")
+    self.class.node_icon_cache[icon] ||=
```

Comment:
Yup, you are right.  I was thinking about this incorrectly when I did it thinking that it would.  I will make a setter method as well.

I would like to take this moment to redirect your attention back to `Fig(1)` in the pull request description.